Calculator/Calculator/Controller/CalculatorViewController.swift

```diff
@@ -7,181 +7,122 @@
 import UIKit
 
 class CalculatorViewController: UIViewController {
-    private var stackCalculation = ""
-    private var finalCalculation = ""
-    private var result = 0.0
-    private var formula: Formula?
-    private var isCalculateResult: Bool = false
+    private var finalCalculation = NameSpace.emptyString
+    private var isCalculateEnd: Bool = false
     
     @IBOutlet weak var calculatorScrollView: UIScrollView!
     @IBOutlet weak var calculatorStackView: UIStackView!
     @IBOutlet weak var operandLabel: UILabel!
     @IBOutlet weak var operatorLabel: UILabel!
     
-    @IBAction func acButton(_ sender: UIButton) {
+    override func viewDidLoad() {
+        super.viewDidLoad()
         resetLabelText()
         resetCalculation()
+        resetCalculatorStackView()
     }
     
-    @IBAction func ceButton(_ sender: UIButton) {
-        stackCalculation = ""
-        operandLabel.text = ""
-    }
-    
-    @IBAction func changeSignButton(_ sender: UIButton) {
-        guard let operandLabelText = operandLabel.text else { return }
+    @IBAction func inputOperandButton(_ sender: UIButton) {
+        if isCalculateEnd == true {
```

Comment:
isCalculateEnd 가 옵셔널이 아니라 아래와 같이 해줘도 좋을 것 같아요 ㅎㅎ
```suggestion
        if isCalculateEnd {
```

Reply:
if 뒤의 condition을 사용할 때 항상 생각해보게 되는 문제같은데 변수명이 변수 뒤에 == true, 혹은 false를 써주어야 할지. 아니면 아무것도 쓰지 않거나 앞에 `!`만 붙여줄지 고민하게 되는 것 같습니다. 
개인적으로는 condition이 참일 경우엔 제안해주신 것처럼 isCalculateEnd라고만 써도 이해가 잘 될 것 같고 반대로 거짓일 경우를 체크할 때에는 뒤에 == false를 붙여주는게 코드를 보는 사람이 이해하기 쉽다고 생각하는데 이 부분에 대해 어떻게 생각하시는지 궁금합니다!

추가로 `옵셔널이 아니라`라는 이유를 들어주셨는데 옵셔널일 경우  == 으로 비교해주면 자동으로 바인딩이 되기 때문에 그렇게 말씀해주신거라고 생각해도 될까요?